index.bs

```diff
@@ -6181,6 +6187,127 @@ The [=remote end steps=] with |command parameters| are:
+timezone on the given top-level traversables or user contexts.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl" data-cddl-module="remote-cddl">
+      emulation.SetTimezoneOverride = (
+        method: "emulation.setTimezoneOverride",
+        params: emulation.SetTimezoneOverrideParameters
+      )
+
+      emulation.SetTimezoneOverrideParameters = {
+        timezone: text / null,
+        ? contexts: [+browsingContext.BrowsingContext],
+        ? userContexts: [+browser.UserContext],
+      }
+    </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+    <code>
+     EmptyResult
+    </code>
+   </dd>
+</dl>
+
+<div algorithm="updated SystemTimeZoneIdentifier steps">
+The [=SystemTimeZoneIdentifier=] algorithm is implementation defined. A
+WebDriver-BiDi [=remote end=] must have an implementation that runs the following
+steps:
+
+1. Let |emulated timezone| be null.
+
+1. Let |realm| be [=current Realm Record=].
+
+1. Let |environment settings| be the [=environment settings object=] whose
+   [=realm execution context=]'s Realm component is |realm|.
+
+1. Let |related navigables| be the result of [=get related navigables=] given
+   |environment settings|.
+
+1. For each |navigable| of |related navigables|:
+
+    1. Let |top-level traversable| be |navigable|’s
+       [=navigable/top-level traversable=].
+
+    1. Let |user context| be |top-level traversable|'s [=associated user context=].
+
+    1. If [=timezone overrides map=] [=map/contains=] |top-level traversable|, set
+       |emulated timezone| to [=timezone overrides map=][|top-level traversable|].
+
+    1. Otherwise, if [=timezone overrides map=] [=map/contains=]
+       |user context|, set |emulated timezone| to
+       [=timezone overrides map=][|user context|].
+
+1. If |emulated timezone| is not null, return |emulated timezone|.
+
+1. Return the result of implementation-defined steps in accordance with the
+   requirements of the [=SystemTimeZoneIdentifier=] specification.
+
+</div>
+
+<div algorithm="remote end steps for emulation.setTimezoneOverride">
+
+The [=remote end steps=] with |command parameters| are:
+
+1. If |command parameters| [=map/contains=] "<code>userContexts</code>"
+   and |command parameters| [=map/contains=] "<code>context</code>",
+   return [=error=] with [=error code=] [=invalid argument=].
+
+1. If |command parameters| doesn't [=map/contain=] "<code>userContexts</code>"
+   and |command parameters| doesn't [=map/contain=] "<code>context</code>",
+   return [=error=] with [=error code=] [=invalid argument=].
+
+1. Let |emulated timezone| be null.
+
+1. If |command parameters| [=map/contains=] "<code>timezone</code>":
+
+   1. Set |emulated timezone| to |command parameters|["<code>timezone</code>"].
+
+   1. If [=IsTimeZoneOffsetString=](|emulated timezone|) returns false and
+      [=AvailableNamedTimeZoneIdentifiers=] does not [=list/contain=]
+      |emulated timezone|, return [=error=] with [=error code=] [=invalid argument=].
```

Comment:
Just to make sure I understood it right, does it mean that we want to support here named timezones (e.g., `Europe/Berlin`) and timezone offsets (e.g., `-120`).